deltachat-rpc-client/src/deltachat_rpc_client/account.py

```diff
@@ -28,7 +28,7 @@ def __eq__(self, other) -> bool:
         return self.id == other.id and self.manager == other.manager
 
     def __ne__(self, other) -> bool:
-        return not self == other
+        return self != other
```

Comment:
this causes an `RecursionError: maximum recursion depth exceeded in comparison`
you can't use the `!=` operator inside the implementation of the `!=` for the class!! I guess the ruff is not that smart if it recommended this change :thinking:

Reply:
to make more explicit the intention to re-use the __eq__ implementation, maybe call it directly instead of `==` but I guess that would be ugly and not "pythonic"

Reply:
Just a heads up: this will be fixed in tonight's release (https://github.com/charliermarsh/ruff/pull/2001)